test/unit/datetime.c

```diff
@@ -177,7 +179,8 @@ tostring_datetime_test(void)
 	};
 	size_t index;
 
-	plan(17);
+	const unsigned tap_tests_per_iter = 1;
+	plan(tap_tests_per_iter * lengthof(tests));
```

Comment:
What's the point of declaring a variable equal to one? To know that a number multiplied by one is the number itself?